swift/Hash Maps and Sets/pairSumUnsorted.swift

```diff
@@ -0,0 +1,14 @@
+func pairSumUnsorted(nums: [Int], target: Int) -> [Int] {
+    var dictionary: [Int: Int] = [:]
```

Comment:
Is "hashmap" a reserved word in swift?
<img width="1408" height="572" alt="image" src="https://github.com/user-attachments/assets/5d9aa846-265f-49ed-b5ad-b5644a49d2b1" />